### ISSUE TITLE
The use of Action-specific min-max normalization in PI0Fast Encoding

### ISSUE THREAD

**Issue by harshas123** [huggingface/lerobot#1531]
Reference: 
1. https://github.com/huggingface/lerobot/blob/862a4439ea4ce4671b84b713f28e705d0b6e172b/src/lerobot/policies/pi0fast/modeling_pi0fast.py#L543
2. https://github.com/huggingface/lerobot/blob/main/src/lerobot/policies/pi0fast/modeling_pi0fast.py#L594
Hello, 
While training a Pi0Fast model on a custom dataset, I noticed that the training loss (multi-class cross entropy) was stabilizing around 8. The resulting actions obtained by using `extract_actions` method of the `PI0FAST` model class was emitting invalid action trajectories. Upon closer examination I see the `normalize_actions` function referenced above. This seems like a 1-way door transform which cannot be reversed as the min-max stats are not stored. In a separate notebook I tested that without the `normalize_actions` I am able to encode and decode the actions successfully but with this function the decoding process cannot recover the original action. I am curious to see if this function is required. 

Code snippet to reproduce the issue. 

# Encode to Pali tokens
```
actions_norm = policy.model.normalize_actions(actions)
fast_tokens_norm = policy.model.fast_tokenizer(actions_norm)
pali_tokens_norm = policy.model._act_tokens_to_paligemma_tokens(torch.tensor(fast_tokens_norm).to(device))
```

# Decode from Pali tokens
```
fast_tokens_norm_hat = policy.model._act_tokens_to_paligemma_tokens(pali_tokens_norm)
decoded_actions_norm = policy.model.fast_tokenizer.decode(fast_tokens_norm_hat)

assert np.allclose(decoded_actions_norm, actions_norm, atol=1e-1), "Normalized Action != Encoded->Decoded Normalized Action"
assert np.allclose(decoded_actions_norm, actions), "Ground truth Action not equal to Encoded-->Decoded Action."
```

**Comment by KDeng98** [comment]
Hi @harshas123, I am also experiencing a similar issue. My fine-tuned model always gives me an extremely wild and jittering motion (even on the push T task). I am wondering how did you solve this problem? Thanks.

**Comment by imstevenpmwork** [comment]
PI0Fast has been deprecated in https://github.com/huggingface/lerobot/pull/2203
Please prioritize the use of the other PI0 family models
